tuning/data/data_preprocessing_utils.py

```diff
@@ -29,6 +32,9 @@ def get_data_collator(
     is_traindata_tokenized: bool,
     max_seq_length: int,
     instruction_template: Optional[str],
+    text_field_name: Optional[str],
+    image_field_name: Optional[str],
+    processor=None,
```

Comment:
I am assuming with the term `processor` you mean [this](https://huggingface.co/docs/transformers/en/main_classes/processors) which is the new name for both image processor and tokenizers in the new HF code. if so can you please update to remove the term `tokenizer`? if we would prefer to have a single argument in that regard.

Reply:
@dushyantbehl I am not adding this change currently since although `AutoProcessor.from_pretrained()` can load both tokenizers and processors, it's hard to distinguish between which one is loaded and then if the processor with tokenizer is loaded then all times where tokenizer is used needs to be replaced with `processor.tokenizer`. Thus since tokenizer and processor are distinguished apart from each other I am keeping the tokenizer term for now